src/components/Common.js

```diff
@@ -12,23 +12,25 @@ export const Card = styled.div`
   ${CardLike};
 `
 
-export const Button = styled.button`
+export const Button = styled.a`
   ${text};
-  cursor: pointer;
+  display: block;
+  text-decoration: none;
   font-family: 'HK Grotesk';
+  font-weight: 600;
```

Comment:
Let's try to use theme here? `${p => p.theme.typography.h2.weight}` is 600

Reply:
m e m e d yessir